hubmap/frontend/src/data/searchfacets.json

```
@@ -0,0 +1,44 @@
+{
+    "facets":
+    {
+        "Donor":{
+            "Genus species": ["Homo Sapiens", "Mus Musculus"],
+            "Organism Age": ["Unspecified", "<0", "0", "1", "2", "3", "4", "5", "6-15", "16-20", "21-30", "31-40", 
+                "41-50", "51-60", "61-70", "71-80", ">80"],
+            "Organism Age unit": ["Unspecified", "day", "week", "month", "year"],
+            "Biological Sex": ["female", "male", "Unspecified"]
+        },
+        "TissueType":{
+            "Sample": ["cell_lines", "organoids", "specimens"],
+            "Organ": ["Unspecified", "lung", "kidney", "bladder", "blood vessel", "bone marrow", "colon", "breast", "liver", "pancreas", "spleen", "thymus", "tonsils"],
+            "Organ Part": ["Unspecified", "lobe of lung", "cortex of kidney", "right lung lobe"],
+            "Model Organ": ["Unspecified", "kidney", "lung"],
+            "Selected Cell Type": ["Unspecified", "kidney cell"]
+        },
+        "Specimen":{
+            "Sample": ["Unspecified", "end stage renal failure"]
+        },
+        "Assays":{
+            "scRNAseq":["snDropseq", "scRNAseq","IHC","LRET-IF","PER-DEI"],
```

```suggestion
            "scRNAseq": ["snDropseq", "scRNAseq", "IHC", "LRET-IF", "PER-DEI"],
```
Here and elsewhere, please whitespace appropriately. Your IDE may be able to help.